app/assets/stylesheets/_pages/_stacks.scss

```diff
@@ -49,6 +49,15 @@
       padding-left: 30px;
       margin-left: -30px;
     }
+    .archived {
+      // From FontAwesome, license @ https://fontawesome.com/license
```

Comment:
This leads to the private license, did you mean to link to https://fontawesome.com/license/free ?

I'd like to be sure there's no licensing issue here.

Reply:
Sorry, yes. Clarified. They say this:

> Downloaded Font Awesome Free files already contain embedded comments with sufficient attribution, so you shouldn't need to do anything additional when using these files normally.

But that's incorrect, the SVG doesn't include a comment.